src/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,602 @@
+        background-image: url('../img/angle-arrow-down.svg');
+        background-position: 88% 44%;
+      }
+    }
+
+    &[aria-expanded=true] {
+      background-image: url('../img/minus-alt.png');
+      background-image: url('../img/minus-alt.svg');
+      background-repeat: no-repeat;
+      background-position: right 0 center;
+      background-size: 1rem;
+
+      @include media($nav-width) {
+        background-color: $color-primary-darkest;
+        color: $color-white;
+        background-image: url('../img/angle-arrow-down-hover.png');
+        background-image: url('../img/angle-arrow-down-hover.svg');
+        background-position: 88% 44%;
+
+        &:hover {
+          background-color: $color-primary-darkest;
+        }
+
+        span {
+          @include nav-border;
+        }
+      }
+    }
+  }
+}
+
+.usa-header-extended {
+  .usa-header {
+    @include media($nav-width) {
+      border-bottom: none;
+    }
+  }
+
+  .usa-logo {
+    @include media($nav-width) {
+      @include margin(3rem null);
+    }
+  }
+
+  em {
+    @include media($nav-width) {
+      font-size: 3.2rem;
+      line-height: 1;
+    }
+  }
+
+  .usa-navbar {
+    @include media($nav-width) {
+      @include outer-container();
+      @include padding(null $site-margins);
+      display: block;
+      height: auto;
+      max-width: $site-max-width;
+    }
+  }
+
+  .usa-nav {
+    @include media($nav-width) {
+      border-bottom: 1px solid $color-gray-light;
+      border-top: 1px solid $color-gray-light;
+      padding: 0;
+      width: 100%;
+    }
+  }
+
+  .usa-nav-inner {
+    @include media($nav-width) {
+      @include outer-container();
+      @include padding(null $site-margins null 2rem);
+      margin-top: -1px;
+      max-width: $site-max-width;
+      position: relative;
+    }
+  }
+
+  .usa-nav-primary {
+    @include media($nav-width) {
+      padding-right: $site-max-width;
+    }
+
+    button {
+      &[aria-expanded=false],
+      &[aria-expanded=true] {
+        @include media($nav-width) {
+          background-position: 88% 50%;
+        }
+      }
+    }
+  }
+
+  .usa-nav-link {
+    @include media($nav-width) {
+      padding-top: 1.9rem;
+    }
+  }
+
+  .usa-nav-submenu {
+    .usa-grid-full {
+      @include media($nav-width) {
+        padding-left: 1.2rem;
+      }
+    }
+  }
+}
+
+.usa-nav-secondary {
+  @include media($nav-width) {
+    float: right;
+    margin-top: -57px;
+  }
+
+
+  .usa-search {
+    @include margin(3rem null);
+
+    @include media($nav-width) {
+      @include margin(-59px null 0 0);
+      display: none;
+      float: left;
+
+      &.is-visible {
+        display: block;
+      }
+    }
+  }
+}
+
+.usa-nav-secondary-links {
+  li {
+    @include media($nav-width) {
+      display: inline;
+      padding-left: 1rem;
+    }
+
+    &:not(:last-child)::after {
+      @include media($nav-width) {
+        content: '|';
+        padding-left: 1rem;
+      }
+    }
+  }
+
+  a,
```

Comment:
I think it should: the container "secondary links" implies everything inside is a "link." But if you can think of a good reason not to and another approach, let me know.